### ISSUE TITLE
Merge & clean up branches

### ISSUE THREAD

**Issue by rtertiaer** [micro-nova/AmpliPi#476]
We have a bunch of branches, some of which are stale and some of which need to be up-streamed. Big ticket items here are:

- [x] Test & merge `webapp_2` for our fancy new interface; blocked by #478 
- [x] Every long-lived active branch save for `main` & `develop` ought to be merged into `develop`; I think `streamer-lms` gets regular bugfixes.
- [ ] If a branch is marked as stale, delete or merge it

**Comment by rtertiaer** [comment]
I'm actually going to just call this closed. We even merged & closed `develop`, and all the PRs pointing to it closed themselves automagically. This made our situation a lot cleaner: everyone examined their open PRs, reopened it and pointed to `main` if it had value. More or less, it's a much more sane version of the world now, even with lots of branches open.
